dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -313,7 +314,6 @@ module cs '../modules/cluster-service.bicep' = {
   }
   dependsOn: [
     maestroServer
-    svcCluster
```

Comment:
this is just a linting fix. the new bicep version complains rightfully about this being unnecessary